server/grpc/server_test.go

```diff
@@ -44,10 +44,12 @@ func (s *IntegrationTestSuite) SetupSuite() {
 
 	s.cfg = network.DefaultConfig()
 	s.cfg.NumValidators = 1
-	s.network = network.New(s.T(), s.cfg)
-	s.Require().NotNil(s.network)
 
-	_, err := s.network.WaitForHeight(2)
+	var err error
+	s.network, err = network.New(s.T(), s.T().TempDir(), s.cfg)
```

Comment:
Could we add a changelog entry? This is an API breaking change. Not sure we should still merge this right now since we have a 0.43 beta freeze

Reply:
[updated changelog](https://github.com/cosmos/cosmos-sdk/pull/9246/commits/4587e1cc6307b874cb19cfb9d009403828f75247) and makes sense to wait